src/main/drivers/max7456.c

```diff
@@ -581,11 +594,11 @@ void max7456DrawScreen(void)
             videoSense = max7456Send(MAX7456ADD_STAT, 0x00);
             DISABLE_MAX7456;
 
-#ifdef DEBUG_MAX7456_SIGNAL
-            debug[0] = videoSignalReg & VIDEO_MODE_MASK;
-            debug[1] = videoSense & 0x7;
-            debug[3] = max7456GetRowsCount();
-#endif
+            if (debugMode == DEBUG_MAX7456_SIGNAL) {
```

Comment:
Not sure if the conditional block is needed, I'd expect gcc to optimise the successive if statements into one.

Reply:
Ack.

Reply:
Are you going to change this now, @jflyper? ;-)

Reply:
Oops...

Reply:
It's not going away :cry:

Reply:
Did it?